### ISSUE TITLE
Error in step "CREATE S3 BUCKET"

### ISSUE THREAD

**Issue by liquidat** [ansible/workshops#280]
##### SUMMARY

The current devel branch fails when a login page is supposed to be created:

```
TASK [aws_workshop_login_page : CREATE S3 BUCKET] ********************************************************************************************************************************************
fatal: [localhost]: FAILED! => 
  msg: 'The conditional check ''result.rc == 0'' failed. The error was: error while evaluating conditional (result.rc == 0): ''dict object'' has no attribute ''rc'''
```
The problem is this piece of code:
 https://github.com/ansible/workshops/blob/7c5b195a31665490126fecb24b3a6d87f08958da/provisioner/roles/aws_workshop_login_page/tasks/main.yml#L16

There is no `rc` in the result dict:
```
  result:
    changed: true
    failed: false
    name: rwolters-security-review.rhdemo.io
    policy: null
    requester_pays: false
    tags: {}
    versioning:
      MfaDelete: Disabled
      Versioning: Disabled
```

##### ISSUE TYPE

 - Bug Report

##### EXTRA VARS FILE

Important bit of `my_vars.yml`:

```
create_login_page: true
```

##### ANSIBLE VERSION

```paste below
ansible 2.8.0
  config file = /home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg
  configured module search path = ['/home/rwolters/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.7.3 (default, May 11 2019, 00:38:04) [GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]
```

##### CONFIGURATION

```paste below
DEFAULT_CALLBACK_WHITELIST(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = ['time']
DEFAULT_FORKS(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = 50
DEFAULT_HOST_LIST(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = ['/home/rwolters/gits/github/liquidat/workshops/provisioner/hosts']
DEFAULT_NO_TARGET_SYSLOG(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = False
DEFAULT_SCP_IF_SSH(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = True
DEFAULT_STDOUT_CALLBACK(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = yaml
HOST_KEY_CHECKING(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = False
INTERPRETER_PYTHON(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = auto_silent
PERSISTENT_COMMAND_TIMEOUT(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = 60
PERSISTENT_CONNECT_TIMEOUT(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = 60
RETRY_FILES_ENABLED(/home/rwolters/gits/github/liquidat/workshops/provisioner/ansible.cfg) = False
```

##### OS / ENVIRONMENT

management node: Fedora 30

##### TOWER

Engine.

##### PLAYBOOK SHORT OUTPUT

```paste below
TASK [include_role : aws_workshop_login_page] ************************************************************************************************************************************************

TASK [aws_workshop_login_page : CREATE STUDENT WEBPAGE] **************************************************************************************************************************************
skipping: [localhost]

TASK [aws_workshop_login_page : CREATE S3 BUCKET] ********************************************************************************************************************************************
fatal: [localhost]: FAILED! => 
  msg: 'The conditional check ''result.rc == 0'' failed. The error was: error while evaluating conditional (result.rc == 0): ''dict object'' has no attribute ''rc'''

PLAY RECAP ***********************************************************************************************************************************************************************************
localhost                  : ok=7    changed=0    unreachable=0    failed=1    skipped=17   rescued=0    ignored=0   

```

**Comment by medberry** [comment]
Ditto, saw the exact same issue.

**Comment by IPvSean** [comment]
fixed with #281
